src/sourcemaputils.rs

```diff
@@ -278,3 +372,20 @@ impl SourceMapProcessor {
         Ok(())
     }
 }
+
+
+#[test]
+fn test_split_url() {
```

Comment:
In this case what happens is that we completely ignore `sourceMappingURL` anyways and override it with a header so that code will never get a query string.